### ISSUE TITLE
Create Neural Network Architecture

### ISSUE THREAD

**Issue by antonsakhanovych** [declspecl/csi4140#12]
Background: The project requires developing a deep neural network to classify CIFAR-10 images. This story should be done after everything else is implemented.
What needs to be done:
1. Created architecture. Check lectures for architectures.
2. At least one fully-connected layer.
3. At least one convolutional layer.
4. At least one ReLU activation layer.
5. A Softmax layer for output classification

### DoD
1. All specified layers are implemented and used in the architecture.
2. No prohibited PyTorch built-in functions are used in the layer
